app/locale/en.coffee

```diff
@@ -1567,6 +1567,7 @@
     no_classrooms_found: "No classrooms found"
     create_classroom_manually: "Create classroom manually"
     classes: "Classes"
+    serial_number: "No."
```

Comment:
Can we put this in the `user:` section next to the other `certificate_` strings?